aiida/work/utils.py

```diff
@@ -8,101 +8,35 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import contextlib
-from threading import local
 import tornado.ioloop
 
 from aiida.common.links import LinkType
-from aiida.orm.calculation import Calculation
+from aiida.orm.calculation import Calculation, WorkCalculation, FunctionCalculation
 from aiida.orm.data.frozendict import FrozenDict
 
-__all__ = ['ProcessStack']
+__all__ = []
 
 PROCESS_STATE_CHANGE_KEY = 'process|state_change|{}'
 PROCESS_STATE_CHANGE_DESCRIPTION = 'The last time a process of type {}, changed state'
+PROCESS_CALC_TYPES = (WorkCalculation, FunctionCalculation)
 
 
-class ProcessStack(object):
+def is_process_calc_type(calc_node):
```

Comment:
I am not sure if this is the best nomenclature. Anything that is run now will be wrapped in a `Process` right? A `JobCalculation` will have an associated `JobProcess`. So inline and job calculations are in a sense also process calcs. I think that using the term `process_calc` to only refer to the `FunctionCalculation` and `WorkCalculation` node types is confusing. To be honest I do not currently have a better name for it, however, given that the utility function is now only used in two places, maybe it is better to scrap it for now. Until we rework the link types and come up with good terminology to differentiate between the "calculation" and "work" planes.